.pre-commit-config.yaml

```diff
@@ -37,7 +37,10 @@ repos:
     rev: v0.8.0
     hooks:
       - id: flake8-ros
+      - id: prettier-xacro
+      - id: prettier-launch-xml
       - id: prettier-package-xml
```

Comment:
Could you check if the change of this file is okay.

Reply:
Looks good! Thank you.